docs/source/en/main_classes/pipelines.mdx

```diff
@@ -25,11 +25,8 @@ There are two categories of pipeline abstractions to be aware about:
   - [`AudioClassificationPipeline`]
   - [`AutomaticSpeechRecognitionPipeline`]
   - [`ConversationalPipeline`]
-<<<<<<< HEAD
-  - [`DocumentQuestionAnsweringPipeline`]
-=======
   - [`DepthEstimationPipeline`]
->>>>>>> Add initial files for depth estimation pipelines
```

Comment:
This was a leftover from a bad rebase I think, I removed those lines and put back the documentation in alphabetical order.